### ISSUE TITLE
Uncaught SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by wgerven** [ng-lightning/ng-lightning#300]
I am really eager to use your library for implementation in my project. I followed the Getting started line by line. My build runs fine (with webpack), but when I launch my app, nothing is rendered and the following error shows: 
![screenshot_2017-01-23_19-02-23](https://cloud.githubusercontent.com/assets/20610101/22216236/8b9cf844-e19e-11e6-92f0-ce35ba669ddc.png)
Any idea where this may come from? 
Thanks!

**Comment by bekos** [comment]
hi @wgerven. Do use use webpack 2? If not you probably want to map `ng-lightning/ng-lightning` to our UMD bundle.

**Comment by wgerven** [comment]
Dear @bekos , thank you for taking the time to help me out. Indeed, I'm not on webpack v2. Honestly, I am a bit of a novice in front-end development. Could you please give me a pointer as to how to map ng-lightning to your UMD bundle? I was looking through your source files, but cannot find anything on it... Thank you. Sorry if this is trivial.

**Comment by bekos** [comment]
No problem :-) In your webpack 1 config you should have something similar to this: 
```js
resolve: {
    alias: {
        'ng-lightning/ng-lightning' : 'ng-lightning/bundles/ng-lightning.umd.js',
    }
  },

```

**Comment by wgerven** [comment]
Thank you @bekos , this however throws me a new error. Again, not when building, but when I open the app
![screenshot_2017-01-24_13-38-31](https://cloud.githubusercontent.com/assets/20610101/22247522/816c8154-e23a-11e6-97ef-cc6315e0296c.png)
My resolve looks as follows
`resolve: {
		alias: {
        'ng-lightning/ng-lightning' : 'ng-lightning/bundles/ng-lightning.umd.js',
    },
    extensions: ['', '.js', '.ts']
  },`
maybe it is not correct?

**Comment by bekos** [comment]
This doesn't seem like an issue in ng-lightning. Take a look here: https://stackoverflow.com/questions/41285889/ctorparameters-map-is-not-a-function-in-angular2-mdl if it can help you.

Probably you need to upgrade your Angular and even better your Webpack :-)

**Comment by wgerven** [comment]
Yeah, I know... I'm just being a bit conservative because of everything you've always got to fix when you upgrade.. Feels always a bit like a house of cards, but I guess that's just my skill! ;)

**Comment by wgerven** [comment]
@bekos , just wanted to let you know that this last issue was resolved by updating to angular 2.4.4.

**Comment by bekos** [comment]
Glad you found a solution! 👍
